src/ext/default.c

```diff
@@ -313,13 +313,13 @@ static double BM25StdNormScorer(const ScoringFunctionArgs *ctx, const RSIndexRes
           dmd->score);
 
   // Normalize the score
-  double normalizedScore = tanhStretched(score, NORMALIZED_BM25_STRETCH_FACTOR);
+  double normalizedScore = tanhStretched(score, RSGlobalConfig.BM25STD_TanhStretchFactor);
 
   // Modify the explanation to include the normalization
   strExpCreateParent(ctx, &scrExp);
   EXPLAIN(scrExp,
     "Final Normalized BM25 : tanh(stretch factor 1/%d * Final BM25 %.2f)",
-    NORMALIZED_BM25_STRETCH_FACTOR, score);
+    RSGlobalConfig.BM25STD_TanhStretchFactor, score);
```

Comment:
query configurations that are used from the background and can be modified after module load should get copied to the query context and read from there, not from the global struct